chatbot/serializers.py

```diff
@@ -11,3 +11,10 @@ class Meta:
         model = Muser
         fields = ('id', 'username', 'gender', 'birthdate', 'evaluations', )
         read_only_fields = ('username', )
+
+
+class MessageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Message
+        fields = ('id', 'created', 'sender', 'receiver', 'text', 'music', 'chips', )
+        ordering = ('created', )
```

Comment:
```suggestion
        ordering = ('created', )

```
add new line at the end of the file

Reply:
glgl sork ej QkffkTwlfhd